.github/workflows/cpu_reusable.yml

```diff
@@ -0,0 +1,39 @@
+name: CPU Reusable workflow
+
+on:
+  workflow_call:
+    inputs:
+      extra_envs:
+        required: false
+        type: string
+      exec_command:
+        required: true
+        type: string
+
+jobs:
+  CPU:
+    # The type of runner that the job will run on
+    runs-on: [self-hosted, cpu]
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Runs a single command using the runners shell
+    - name: Checkout
+      uses: actions/checkout@v2.4.0
+    - name: Build Dev Docker
+      run: |
+        set -e
+        git submodule sync
+        git submodule update --depth=1 --init --recursive
+        docker build -t disc-dev-cpu --build-arg BASEIMAGE=nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04 -f docker/dev/Dockerfile .
```

Comment:
CPU test with cuda image? is this still the issue introduced by TF ?

Reply:
Nope, the workflow configuration was committed previously. I refactor it into a reusable workflow in order to share it between TensorFlow and PyTorch.